app/views/admin/delayed_jobs/_row.haml

```
@@ -0,0 +1,63 @@
+- describe = JobDetail.new(job)
+%tr
+  %td
+    %strong Job ID:
+    = job.id
+    %br
+    %strong Job Name:
+    = describe.job_name
+
+    - if describe.user_id
+      %br
+      %strong Run by User:
+      = users[describe.user_id] || describe.user_id
+
+    - if describe.job_class
+      %br
+      %strong Job Item:
+      = describe.job_class
+      -# only useful if we have a job class
+      - if describe.report_id
+        %br
+        %strong Report ID:
+        = describe.report_id
+
+      - if describe.created_at
+        %br
+        %strong Item Originally Requested:
+        = describe.created_at
+    %br
+    %strong Run After:
+    = job.run_at
+    - if job.locked_by
+      %br
+      %span.icon-cog.text-success
+      %strong Locked by
+      = job.locked_by
+      at
+      = job.locked_at
+    - if job.failed_at
+      %br
+      %span.icon-cancel-circled.text-danger
+      %strong Failed:
+      = job.failed_at
+    - if job.cancellation_requested_at
+      %br
+      %span.icon-cancel-circled.text-warning
+      %strong Cancellation Requested:
+      = job.cancellation_requested_at
+  %td= job.queue
+  %td= job.priority
+  - if can_manage_config?
```

this is new, hiding the error for non-admins